webbpsf/webbpsf_core.py

```diff
@@ -1564,6 +1720,10 @@ def show_notebook_interface(instrument):
     from IPython.display import display, clear_output
     from matplotlib import pyplot as plt
 
+
+    if isinstance(instrument, string_types):
```

Comment:
OK, I've written a `show_notebook_interface()` top-level function which takes a string input of any instrument name, then behind the scenes creates the instrument instance and calls either `show_notebook_interface_wfi()` or `show_notebook_interface_jwst()`. All of this exists in a new `jupyter_gui.py` file to keep things organized.